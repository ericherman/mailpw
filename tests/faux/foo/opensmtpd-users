ada	$1$COzMUgHH$zcbfWM62RDM6wq2yJOu/E1
brian	$1$T2W2ATor$d4J36Zy.uS.tYWHH04wi31
USER	$1$jQBKBDX9$PkZYddnr5iePbrU4tMF.P.
don	$1$WezNzVpM$JmbHh5T.nHeioVj/c9Yqh1
