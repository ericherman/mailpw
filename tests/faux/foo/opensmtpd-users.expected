ada	$1$COzMUgHH$zcbfWM62RDM6wq2yJOu/E1
brian	$1$T2W2ATor$d4J36Zy.uS.tYWHH04wi31
USER	$6$pinch.of.salt$7Jf16S36pj3hnNd7YxvqugbtIn6ZQi3tEWSv7wtKm.lQJp7mphib8UkrpN6ME4tDrj4vmuryXsrVDuse/4HS5.
don	$1$WezNzVpM$JmbHh5T.nHeioVj/c9Yqh1
